recipe/bld.bat

```diff
@@ -5,3 +5,12 @@ if errorlevel 1 exit 1
 nmake /f makefile.vc install
 if errorlevel 1 exit 1
 
+nmake /f makefile.vc clean
+if errorlevel 1 exit 1
+
+nmake /f makefile_mod.vc
+if errorlevel 1 exit 1
+
+nmake /f makefile_mod.vc install
+if errorlevel 1 exit 1
```

Comment:
Okay, I hope maybe I've figured it out.  The second build wasn't running on post (maybe not any) files because the `.obj` files exist from the previous build.  But we need them to be rebuilt with the `\DLOADABLE_EXTENSION` flag, so I think we need to make clean first.

(*fingers crossed*)